server/portal/apps/workspace/api/views.py

```diff
@@ -281,7 +283,7 @@ def post(self, request, *args, **kwargs):
 
         if job_uuid and job_action:
             if job_action == 'resubmit':
-                METRICS.info("user:{} is resubmitting job uuid:{}".format(username, job_uuid))
+                logger.info("user:{} is resubmitting job uuid:{}".format(username, job_uuid))
```

Comment:
`METRICS.info` now expects a certain set of values to be passed used for audit trails, so we can use `logger.info` when logging non-audit-trails logs